cmd/krel/cmd/gcbmgr.go

```diff
@@ -269,23 +316,33 @@ func setGCBSubstitutions() (map[string]string, error) {
 	// TODO: Remove once we remove support for --built-at-head.
 	gcbSubs["BUILD_AT_HEAD"] = ""
 
-	buildpoint := gcbmgrOpts.buildVersion
+	buildpoint := o.buildVersion
 	buildpoint = strings.ReplaceAll(buildpoint, "+", "-")
 	gcbSubs["BUILD_POINT"] = buildpoint
 
 	// TODO: Add conditionals for find_green_build
-	buildVersion := gcbmgrOpts.buildVersion
+	buildVersion := o.buildVersion
 	buildVersion = fmt.Sprintf("--buildversion=%s", buildVersion)
 	gcbSubs["BUILDVERSION"] = buildVersion
 
-	if gcbmgrOpts.branch != "" {
-		gcbSubs["RELEASE_BRANCH"] = gcbmgrOpts.branch
+	branch := o.branch
+
+	if branch != "" {
+		gcbSubs["RELEASE_BRANCH"] = branch
 	} else {
 		return gcbSubs, errors.New("Release branch must be set to continue")
 	}
```

Comment:
Instead of `if/else` I think better approach would be to do:
```go
gcbSubs["RELEASE_BRANCH"] = branch

if branch == "" {
	return gcbSubs, errors.New("Release branch must be set to continue")
}
```
it's more clear and concise what is the expected behaviour

Reply:
Addressed in https://github.com/kubernetes/release/pull/1104.
Just a note that the map assignment should happen after the check, but agreed that this is cleaner.